README.md

```diff
@@ -128,10 +124,9 @@ Note: Option boot using a boot screen requires a keyboard in directly into the m
 2. Because the preinstall script runs in memory, do not attempt run the preinstall twice in the same boot session. This will cause major delays when opening the Big Sur install app/task.
 
 ### How to turn off System Integrity Protection (this is now built into preinstall.sh)
-1. Open Terminal in the booted recovery disk (and possibly external USB Big Sur USB installer disks made with createinstallmedia).
-2. Execute `csrutil disable`.
-3. Execute `csrutil authenticated-root disable` (can only be done from Big Sur Recovery disks).
-4. Use Start up disk (top left to select your installation).
+1. Open Terminal in the booted recovery disk (and possibly external USB Big Sur USB installer disks made with `createinstallmedia`).
+2. Execute `csrutil disable; csrutil authenticated-root disable` (can only be done from Big Sur Recovery disks).
```

Comment:
do not combine these commands. Only csrutil authenticated-root disable needs to be done on Big Sur recovery or USB Installer disks.

Reply:
Both `csrutil disable` and `csrutil authenticated-root` needs to be executed from the Recovery OS.